### ISSUE TITLE
Generated WSDL reply uses a namespace before declaring it

### ISSUE THREAD

**Issue by YaroslavKormushyn** [DigDes/SoapCore#190]
**TL;DR**: Using the DataContractSerializer I'm unable to use Guid type properties in input models.

I'm using 0.9.9.3 and have troubles generating a client from WSDL.
I'm also using the Visual Studio WCF Test Client.

Input object:
```
[DataContract]  
public class Model  
{
    [DataMember]
    public Guid Id { get; set; }
}
```

Service contract:
```
[ServiceContract]
public interface IServiceContract
{
    [OperationContract]
    int GetItem(Model model);
}
```

Service itself:
```
public class SampleService : IServiceContract
{
    public int GetItem(Model model) => 0;
}
```

Endpoint registered as:
`app.UseSoapEndpoint<IServiceContract>("/SampleService.svc", new BasicHttpBinding());`

The generated WSDL defines the guid type in the `tns` namespace, but uses the guid type from `ser` namespace, which is defined after the input model. Because of that the WCF Test Client cannot generate a request.

Error output:
`Error: Cannot obtain Metadata from https://localhost:44342/SampleService.svc If this is a Windows (R) Communication Foundation service to which you have access, please check that you have enabled metadata publishing at the specified address.  For help enabling metadata publishing, please refer to the MSDN documentation at http://go.microsoft.com/fwlink/?LinkId=65455.WS-Metadata Exchange Error    URI: https://localhost:44342/SampleService.svc    Metadata contains a reference that cannot be resolved: 'https://localhost:44342/SampleService.svc'.    The server did not provide a meaningful reply; this might be caused by a contract mismatch, a premature session shutdown or an internal server error.HTTP GET Error    URI: https://localhost:44342/SampleService.svc    The document at the url https://localhost:44342/SampleService.svc was not recognized as a known document type.The error message from each known type may help you fix the problem:
- Report from 'DISCO Document' is 'Discovery document at the URL https://localhost:44342/SampleService.svc could not be found.'.  
- The document format is not recognized.
- Report from 'WSDL Document' is 'There is an error in XML document (1, 4165).'.  
- Namespace prefix 'ser' is not defined.
- Report from 'XML Schema' is 'The root element of a W3C XML Schema should be <schema> and its namespace should be 'http://www.w3.org/2001/XMLSchema'.'.`

When using XmlSerializer the problem seems to be solved, but this introduces a new problem with serializing IEnumerable properties (which XmlSerializer cannot serialize, this is a known limitation).

Is it how it's supposed to work and I'm missing some cumbersome configuration or is it a bug?

Generated WSDL attached.
[SampleService.zip](https://github.com/DigDes/SoapCore/files/2841378/SampleService.zip)

**Comment by kotovaleksandr** [comment]
@YaroslavKormushyn Please check on latest sources

**Comment by KamranShahid** [comment]
> @YaroslavKormushyn Please check on latest sources

Getting similar response
